src/core/lib/security/credentials/jwt/jwt_verifier.cc

```diff
@@ -535,6 +540,7 @@ static EVP_PKEY* pkey_from_jwk(const Json& json, const char* kty) {
     gpr_log(GPR_ERROR, "Unsupported key type %s.", kty);
     goto end;
   }
+#if OPENSSL_VERSION_NUMBER < 0x30000000L
```

Comment:
This doesn't have a matching `#endif` directive

I believe the right thing to do is change the `#if` immediately below this to an `#elif`